tests/ui/test_mcg_ui.py

```diff
@@ -0,0 +1,271 @@
+import logging
+from ocs_ci.ocs import constants
+from ocs_ci.helpers.helpers import create_unique_resource_name
+import time
+
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import skipif_ocs_version, skipif_disconnected_cluster, ui
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.ui.mcg_ui import BucketClassUI, MCGStoreUI, ObcUI
+from ocs_ci.ocs.utils import (
+    oc_get_all_resource_names_of_a_kind,
+)
+from ocs_ci.utility.utils import check_resource_existence
+
+logger = logging.getLogger(__name__)
+
+
+class TestStoreUserInterface(object):
+    """
+    Test the MCG store UI
+
+    """
+
+    def teardown(self):
+        for store_kind in ["namespacestore", "backingstore"]:
+            test_stores = [
+                store_name
+                for store_name in oc_get_all_resource_names_of_a_kind(store_kind)
+                if f"{store_kind}-aws" in store_name
+            ]
+            for store_name in test_stores:
+                OCP(kind=store_kind).delete(resource_name=store_name)
+
+    @ui
```

Comment:
since we don't have jobs for the `ui` tier, these tests should be part of `tierx`